bin/inference/pycbc_inference

```diff
@@ -141,7 +141,12 @@ with ctx:
     if pool.is_main_process():
         for fn in [sampler.checkpoint_file, sampler.backup_file]:
             with loadfile(fn, 'a') as fp:
-                fp.write_config_file(cp)
+                # some models will interally modify original cp for sampling,
```

Comment:
This shouldn't be an option as one should always save the original configuration. Why isn't your internal sampler modifying a copy? The version saved here then doesn't have to know (and really shouldn't) that you may have modified internally.

Reply:
@ahnitz My understanding is `cp` is saved when it runs `logging.info("Loading joint_primary_marginalized model")
        return super(HierarchicalModel, cls).from_config(
                cp, submodels=submodels, **kwargs)` https://github.com/WuShichao/pycbc/blob/accelerate_multiband/pycbc/inference/models/hierarchical.py#L1002 the `cp` here is the modified config for sampling, can't be used as the initial config. So how to let PyCBC save the original one here?

Reply:
@WuShichao I don't see anything related to saving the configfile where you've pointed. The place is in pycbc_inference where I'm comment on. Take a look at my first comment, I say how to do it. Don't modify the configparser you are passed in-place. Make a copy so you aren't editting the original. Then you don't need to save a separate copy and this particular line will just work to begin with.